modules/ROOT/pages/_partials/installing-mobile-services.adoc

```diff
@@ -124,14 +124,14 @@ image:catalog-mobile-clients.png[]
 = Additional resources
 
 == Troubleshooting
-If you have problems running {org-name} {product-name}, run the installer using the --debug option to capture as much information as possible:
+If you have problems running {org-name} {product-name}, run the installer using the '--debug' option to capture as much information as possible:
```

Comment:
needs to be ` rather than '

https://www.methods.co.nz/asciidoc/faq.html#_what_8217_s_the_difference_between_quoted_text_and_quoted_monospaced_text